### PR TITLE
CAR-125 added edit/save rules functional

### PR DESCRIPTION
**Pull request by RomanFedytskyi**

### CODE REVIEW

File: dev/publicDev/managers-page/data.service.js [hunk 1]
```diff
@@ -25,9 +26,9 @@
       }
 
       // TODO: future update functionality
```
Comment (dmitrij-borchuk) [inline dev/publicDev/managers-page/data.service.js]: Remove it

File: dev/publicDev/managers-page/managers-controller.js [hunk 2]
```diff
@@ -28,6 +28,10 @@
       vm.newRule = {}
     }
 
+    vm.edit = function(index) {
```
Comment (dmitrij-borchuk) [inline dev/publicDev/managers-page/managers-controller.js]: Do not use 'index', pass 'rule' to the function

File: dev/publicDev/managers-page/managers-controller.js [hunk 3]
```diff
@@ -36,8 +40,9 @@
     }
 
     // TODO: future update functionality
-    vm.edit = function(rule) {
-      rulesDataService.update(rule._id, rule).then(function() {
+    vm.save = function(index, id, rule) {
```
Comment (dmitrij-borchuk) [inline dev/publicDev/managers-page/managers-controller.js]: Do not use 'index', pass 'rule' to the function

File: dev/publicDev/managers-page/style.scss [hunk 4]
```diff
@@ -55,8 +55,8 @@
     padding-left: 20px;
   }
 
-  .new-rule {
-    display: flex;
+  .btn-block+.btn-block {
```
Comment (dmitrij-borchuk) [inline dev/publicDev/managers-page/style.scss]: Do not use '+'

**Review by dmitrij-borchuk — CHANGES_REQUESTED** [review]

File: dev/publicDev/managers-page/data.service.js (hunk 1 above)
Comment (RomanFedytskyi) [reply dev/publicDev/managers-page/data.service.js]: fixed

File: dev/publicDev/managers-page/managers-controller.js (hunk 2 above)
Comment (RomanFedytskyi) [reply dev/publicDev/managers-page/managers-controller.js]: fixed

File: dev/publicDev/managers-page/managers-controller.js (hunk 3 above)
Comment (RomanFedytskyi) [reply dev/publicDev/managers-page/managers-controller.js]: fixed

File: dev/publicDev/managers-page/style.scss (hunk 4 above)
Comment (RomanFedytskyi) [reply dev/publicDev/managers-page/style.scss]: fixed

**Pull request merged**